src/main/app/documents/documentsClient.ts

```diff
@@ -32,12 +35,14 @@ export class DocumentsClient {
     if (StringUtils.isBlank(bearerToken)) {
       throw new Error('User authorisation cannot be blank')
     }
-    return requestNonPromise.get(
```

Comment:
is requestNonPromise still used for anything?

Reply:
No, would you like me to remove it?

Reply:
would make sense to as it was only added for this

Reply:
Will do 👍

Reply:
Done